scipy/stats/_multivariate.py

```diff
@@ -564,24 +564,32 @@ def _cdf(self, x, mean, cov, maxpts, abseps, releps):
             Absolute error tolerance
         releps : float
             Relative error tolerance
+        lower_limit : array_like, optional
+            Lower limit of integration of the cumulative distribution function.
+            Default is negative infinity. Must be broadcastable with `x`.
 
         Notes
         -----
         As this function does no argument checking, it should not be
         called directly; use 'cdf' instead.
 
+
         .. versionadded:: 1.0.0
 
         """
-        lower = np.full(mean.shape, -np.inf)
+        lower = (np.full(mean.shape, -np.inf)
+                 if lower_limit is None else lower_limit)
+        x, lower = np.broadcast_arrays(x, lower)
+        n = x.shape[-1]
+        limits = np.concatenate((lower, x), axis=-1)
         # mvnun expects 1-d arguments, so process points sequentially
-        func1d = lambda x_slice: _mvn.mvnun(lower, x_slice, mean, cov,
+        func1d = lambda limits: _mvn.mvnun(limits[:n], limits[n:], mean, cov,
                                            maxpts, abseps, releps)[0]
-        out = np.apply_along_axis(func1d, -1, x)
+        out = np.apply_along_axis(func1d, -1, limits)
```

Comment:
`apply_along` axis applies `func1d` to every slice along the last axis. So we concatenate both lower and upper integration limits along the last axis and split them into separate lower and upper limits within `func1d`.